sdk-base/src/main/java/com/mapbox/maps/dsl/CameraOptionsKtx.kt

```diff
@@ -24,4 +24,10 @@ inline fun cameraOptions(
   .zoom(cameraState.zoom)
   .pitch(cameraState.pitch)
   .apply(block)
-  .build()
\ No newline at end of file
+  .build()
+
+/**
+ * Check if camera options has all the empty properties.
+ */
+fun CameraOptions.isEmpty() =
+  center == null && bearing == null && pitch == null && padding == null && zoom == null && anchor == null
```

Comment:
does this need to be a public API?

Reply:
will be removed per discussion with @yunikkk - I'll create follow-up ticket later